src/components/pages/gallery/Upload.tsx

```diff
@@ -322,6 +311,8 @@ export default function Upload(props: Props) {
             setProgressView(false);
             throw err;
         } finally {
+            // set state to done
+            await ImportService.setDoneUploadingFiles();
```

Comment:
I fear if there is some error and we don't mark it as done in the `finally` block, it will keep reappearing every time the app starts up. 😅
Should I move it to where it was earlier?

Reply:
ohh okk cool understood

Reply:
but then we would lose the records of the files that failed na? because when we set `done` we clear the `filePaths` array  ?

Reply:
Yeah right

Reply:
we are anyways getting rid of the `done` property so we can ignore this